docker/env/version

```diff
@@ -1 +1 @@
-1.25
+1.28
```

Comment:
next time wait until the final review/merge to take a version, use something like `1.28-hdr-1`

Reply:
i used the 1.22-hdr, but number run to fast